test/signs_ui/signs/messages_test.exs

```diff
@@ -14,4 +14,18 @@ defmodule SignsUi.Signs.MessagesTest do
     assert %{"RDTC-n" => [%{duration: _, text: ""}, %{duration: _, text: "Alewife 12 min"}]} =
              SignsUi.Signs.Messages.list_messages(pid)
   end
+
+  test "when the message has multiple dashes it doesnt crash" do
+    {:ok, pid} = SignsUi.Signs.Messages.start_link()
+
+    SignsUi.Signs.Messages.add_message(pid, %{
+      "MsgType" => "SignContent",
+      "c" => ["e130~n2-\"Alewife 12 min\".4-\"Braintre 4 min\""],
+      "sta" => "RDTC",
+      "uid" => "616722"
+    })
+
+    assert %{"RDTC-n" => [%{duration: _, text: ""}, %{duration: _, text: "Alewife 12 min\".4"}]} =
```

Comment:
Hmm, I don't know about the `\".4"` stuff. I guess it does fix the crash, which is useful, but maybe it should also only show whats in the quotes? I think this can be done pretty easily with a regex that captures what's between the quotes.